src/main/webapp/i18n/de/exam.json

```diff
@@ -1,10 +1,23 @@
 {
     "artemisApp": {
         "exam": {
-            "title": "Exam"
+            "title": "Klausur",
+            "visible": "Sichtbar",
+            "startDate": "Anfang",
```

Comment:
Wouldn't be "Beginn" be a better naming? "Beginn" indicates a point in time whereas "Anfang" indicates more an area in time imo.